test/lit/passes/dae_tnh.wast

```diff
@@ -0,0 +1,35 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; NOTE: This test was ported using port_test.py and could be cleaned up.
```

Comment:
Was this generated with `port_test.py` or that left over from a copy-paste?

Reply:
Ah, heh, that's a copy-paste... good catch.